### PR TITLE
[KIECLOUD-448] - Provide automated way to update product version on R…

### PR DESCRIPTION
**Pull request by spolti**
…HPAM and RHDM repositories

Signed-off-by: spolti <fspolti@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: scripts/update-version.py [hunk 1]
```diff
@@ -0,0 +1,397 @@
+#!/usr/bin/python3
+# This script will to help to manage rhpam components modules version, it will update all needed files
+# Example of usage:
+#   # move the current version to the next one or rcX
+#   python scripts/update-version.py -v 7.15.1
```
Comment (Kaitou786) [inline scripts/update-version.py:5]: change to python3 and maybe mention this is a dry-run and user would need to add `--confirm` to actually change

**Comment by Kaitou786** [comment]
After running the script I see there some unwanted changes:
In README.md:
Screenshot from 2021-01-12 18-10-18
Screenshot from 2021-01-12 18-11-27
Screenshot from 2021-01-12 18-12-08

File: scripts/update-version.py (hunk 1 above)
Comment (spolti) [reply scripts/update-version.py]: sure thing, updated.

**Comment by spolti** [comment]
@Kaitou786 updated.

**Review by msmagnanijr — DISMISSED** [review]
lgtm

**Review by Kaitou786 — APPROVED** [review]

**Pull request merged**